### DISCUSSION TITLE
arm64 编译失败，求助

### DISCUSSION THREAD

**Discussion by vinfai** [apache/doris#9863] [A - General / Q&A]
linux info:
Linux iZib208v4tgiakxrlnwxogZ 4.18.0-80.7.2.el7.aarch64 #1 SMP Thu Sep 12 16:13:20 UTC 2019 aarch64 aarch64 aarch64 GNU/Linux

fail info:
g++-11: error: unrecognized command-line option '-msse4.2'
make[2]: *** [src/CMakeFiles/libhdfs3-shared.dir/client/DataTransferProtocolSender.cpp.o] Error 1
make[2]: Leaving directory `/opt/tools/apache-doris-1.0.0-incubating-src/thirdparty/src/libhdfs3-master/doris_build'
make[1]: *** [src/CMakeFiles/libhdfs3-shared.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
g++-11: error: unrecognized command-line option '-msse4.2'
make[2]: *** [src/CMakeFiles/libhdfs3-static.dir/client/DataTransferProtocolSender.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
[ 13%] Building CXX object src/CMakeFiles/libhdfs3-static.dir/client/DirectoryIterator.cpp.o

请问下大家有怎么改

**Comment by RobinWuDev** [comment]
doris的1.1.0之前的版本不支持arm64，推荐直接使用1.1.0的开始编译，不然即使这个问题解决了，还会有其他的问题。今天我编译了一天，一个问题接着一个问题。

**Reply by dataalive** [reply]
你的解决了吗？

**Reply by RobinWuDev** [reply]
我用1.1.0-preivew的tag，可以编译成功。后面创建docker的启动镜像也没有问题。
